__tests__/app.test.js

```diff
@@ -35,4 +35,17 @@ describe("GET /api/topics", ()=> {
             expect(msg).toBe("Not found: API not found")
         })
     })
-});
\ No newline at end of file
+});
+
+
+describe("GET /api endpoints", ()=> {
+    test("200: responds with the correct endpoints", () => {
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body: { endpoints }}) => {
+            expect(endpoints).toHaveProperty("GET /api");
+            expect(endpoints).toHaveProperty("GET /api/topics");
+            });
```

Comment:
This won't prove that your new endpoints work so you'll have to keep going back to this test and adding to it as you go. 

Instead, to make this dynamic, you could require in the endpoints.json and check that the body returned here looks the same as it 👍